crates/ruff/resources/test/fixtures/pyupgrade/UP036_4.py

```diff
@@ -7,28 +7,28 @@
 
 if True:
     if foo:
-        pass
+        print()
```

Comment:
Why this?

Reply:
i had trouble telling apart the `pass` that were in the input and the `pass` that we insert for removing the last statement in body